client-js/main/client/firebase-subscription-service.js

```diff
@@ -59,19 +58,46 @@ export class FirebaseSubscriptionService {
       throw new Error('This subscription is already registered in subscription service');
     }
     this._subscriptions.push(subscription);
+
+    if (!this._isRunning()) {
+      this._run();
+    }
   }
 
   /**
-   * Starts the subscription service, keeping up the added subscriptions.
+   * Indicates whether this service is running keeping up subscriptions.
+   *
+   * @returns {boolean}
+   * @private
    */
-  run() {
-    if (this._interval) {
-      throw new Error('The FirebaseSubscriptionService is already running');
-    }
+  _isRunning() {
+     return !!this._interval;
+  }
 
+  /**
+   * Starts the subscription service, keeping up the added subscriptions.
+   *
+   * @private
+   */
+  _run() {
     this._interval = setInterval(() => {
       this._keepUpSubscriptions();
-    }, SUBSCRIPTION_KEEP_UP_INTERVAL.inMs());
+    }, this._keepUpInterval());
+  }
+
+  /** Interval in milliseconds for sending subscription keep up requests. */
+  _keepUpInterval() {
```

Comment:
If this method stays unchanged after addressing the @dmdashenkov request, it should be marked `@private` like all others.